### PR TITLE
[CI] use latest Ansible releases (2.4 and 2.5)

### PR DESCRIPTION
**Pull request by pilou-**
CI: use latest Ansible releases (2.4 and 2.5)

### CODE REVIEW

**Comment by pilou-** [comment]
ping @mgu

File: .travis.yml [hunk 1]
```diff
@@ -7,12 +7,12 @@ python:
 
 env:
   matrix:
-  - ANSIBLE_REF=2.4.3.0
-  - ANSIBLE_REF=2.3.3.0
+  - ANSIBLE_REF=2.6
+  - ANSIBLE_REF=2.5
```
Comment (mgu) [inline .travis.yml]: it is not matching the name of your PR, is it OK ? :)

**Comment by pilou-** [comment]
In fact, it is:
- `<2.6` will install 2.5.X.Y (Travis [log](https://travis-ci.org/peopledoc/ansible-boot-lxc/jobs/358869008#L468))
- `<2.5` will install 2.4.X.Y (Travis [log](https://travis-ci.org/peopledoc/ansible-role-oracle-java/jobs/358878199#L468))

File: .travis.yml (hunk 1 above)
Comment (mgu) [reply .travis.yml]: could you change this to `ANSIBLE_REF="ansible<2.6"` here ?
it seems easier to read/understand (to me)

Comment (pilou-) [reply .travis.yml]: Indeed ! Updated:
- 2.5.0: https://travis-ci.org/peopledoc/ansible-boot-lxc/jobs/359266622#L740
- 2.4.3.0: https://travis-ci.org/peopledoc/ansible-boot-lxc/jobs/359266623#L744

**Review by mgu — APPROVED** [review]

**Pull request merged**